src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(5) {
+  margin-right: 0;
+}
+
+.stars--1 div:nth-child(1),
```

```suggestion
.stars--1 :nth-child(1),
```
There is no need to increase the specificity with 'div'